crates/astria-conductor/src/bin/astria-conductor.rs

```diff
@@ -41,17 +30,16 @@ async fn main() -> Result<()> {
 }
 
 async fn run() -> Result<()> {
-    let args = Cli::parse();
+    let conf = match config::get() {
+        Ok(cfg) => cfg,
+        Err(e) => {
+            eprintln!("failed to read configuration: {e}");
+            std::process::exit(2);
+        }
+    };
+
     // hierarchical config. cli args override Envars which override toml config values
-    let conf: Config = Figment::new()
-        .merge(Toml::file("ConductorConfig.toml"))
-        .merge(Env::prefixed("RUST_").split("_").only(&["log"]))
-        .merge(Env::prefixed("ASTRIA_"))
-        .merge(Serialized::defaults(args))
-        .extract()?;
-
-    telemetry::init(std::io::stdout, conf.log.as_deref().unwrap_or("info"))
-        .expect("failed to initialize telemetry");
+    telemetry::init(std::io::stdout, &conf.log).expect("failed to initialize telemetry");
```

Comment:
Since we are in a `run()` function that returns back to `main` we should just `wrap_err("failed to initialize telemtry")?` if we touch this line.